Jenkinsfile

```diff
@@ -58,6 +58,8 @@ END
         try{
             postStatus(getPendingStatus("Running the linters..."))
             sh "docker-compose run --rm -T  webpack npm run eslint"
+            sh "docker-compose run --rm eventkit black --config /var/lib/eventkit/config/pyproject.toml --check /home/eventkit/miniconda3/envs/eventkit-cloud/lib/python3.6/site-packages/eventkit_cloud"
+            sh "docker-compose run --rm eventkit flake8 eventkit_cloud"
```

Comment:
I think we need to point this to the config file, as it stands I do not believe it's using our current config file.  http://flake8.pycqa.org/en/2.6.0/config.html

Reply:
So we'd need to change this to something like `docker-compose run --rm eventkit flake8 --config /var/lib/eventkit/config/setup.cfg eventkit_cloud` I believe.